app/services/slack_client.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+class SlackClient
+  class Error < StandardError
+  end
+
+  def self.create_message(message)
+    new.create_message(message)
+  end
+
+  def create_message(message)
+    response =
+      client.post(
+        'chat.postMessage',
+        { channel: "##{ENV['SLACK_CHANNEL']}", mrkdwn: true, text: message },
+      )
```

Comment:
This uses the Slack API to post a message to a certain channel, which requires us to provision a Slack user with full read/write access.

There's a simpler way using the [Slack webhook API](https://api.slack.com/messaging/webhooks). It has less privileges; it only fires a message into a channel and can't read anything. Creating the Webhook API URL key should also be easier.

Here's how Apply does it: https://github.com/DFE-Digital/apply-for-teacher-training/blob/e85ff42219275d6728efabb225464340b195178d/app/workers/slack_notification_worker.rb#L7

Reply:
Also, we can just hardcode the `SLACK_CHANNEL` and not set it in an environment variable.